src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -24,7 +24,43 @@
 
 package jenkins.branch;
 
-import antlr.ANTLRException;
+import static jenkins.scm.api.SCMEvent.Type.CREATED;
+import static jenkins.scm.api.SCMEvent.Type.UPDATED;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import javax.servlet.ServletException;
+
+import org.acegisecurity.AccessDeniedException;
+import org.acegisecurity.Authentication;
```

Comment:
My IDE alphabetized imports.

`org.acegisecurity.Authentication` is an addition.